### PR TITLE
correct electricity activity_ids

### PR DESCRIPTION
**Pull request by gpantelidou**
**Is this a new addition or an update to an existing factor?**

- [ ] New
- [x] Update

**What category or categories does it affect?**

Electricity

**What?**

correct the electricity activity_ids according to the new proposed structure

**Why?**

correct breaking changes

### CODE REVIEW

File: OpenEmissionFactorsDB.csv [hunk 1]
```diff
@@ -1,6 +1,6 @@
...
-12c67ef6-cd25-4556-8238-62f042937895,Energy,Electricity,electricity-energy_source_gas_fired_plant,Electricity supplied from gas fired plant,kWh,0.418,,,,,,,10,2|3,upstream-fuel_combustion,ADEME,Base Carbone,2021,2021,,FR,,Climatiq,2021/08/20,Emission intensity for gas fired plant. The data scope includes upstream and combustion emissions. Retrieved from Base Carbone - Data V21.0.csv.,https://bilans-ges.ademe.fr/en/accueil/contenu/index/page/downloaddata/siGras/0
+d3046b43-aa23-4684-a6f7-117fa2086a0f,Energy,Electricity,electricity-supply_grid-source_coal_fired_plant,Electricity supplied from coal fired plant,kWh,1.06,,,,,,,10,2|3,upstream-fuel_combustion,ADEME,Base Carbone,2021,2021,,FR,,Climatiq,2021/08/20,Emission intensity for coal fired plant. The data scope includes upstream and combustion emissions. Retrieved from Base Carbone - Data V21.0.csv.,https://bilans-ges.ademe.fr/en/accueil/contenu/index/page/downloaddata/siGras/0
```
Comment (maryarbab) [inline OpenEmissionFactorsDB.csv:2]: source_coal would be sufficient.

File: OpenEmissionFactorsDB.csv [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
+12c67ef6-cd25-4556-8238-62f042937895,Energy,Electricity,electricity-supply_grid-source_gas_fired_plant,Electricity supplied from gas fired plant,kWh,0.418,,,,,,,10,2|3,upstream-fuel_combustion,ADEME,Base Carbone,2021,2021,,FR,,Climatiq,2021/08/20,Emission intensity for gas fired plant. The data scope includes upstream and combustion emissions. Retrieved from Base Carbone - Data V21.0.csv.,https://bilans-ges.ademe.fr/en/accueil/contenu/index/page/downloaddata/siGras/0
```
Comment (maryarbab) [inline OpenEmissionFactorsDB.csv:3]: source_gas would be sufficient.

File: OpenEmissionFactorsDB.csv [hunk 3]
```diff
@@ -151,9 +151,9 @@ c3f5582d-900e-40ca-8bd0-b2bc58964bfb,Energy,Electricity,electricity-supply_grid-
...
-ff48cb32-52f7-481e-8709-3c4d97d5ea87,Energy,Electricity,electricity-energy_source_nuclear_plant,Electricity supplied from nuclear plant,kWh,0.006,,,,,,,10,3,upstream,ADEME,Base Carbone,2021,2021,,FR,,Climatiq,2021/11/30,Emission intensity for nuclear power plant. The data scope includes upstream emissions. Retrieved from Base Carbone - Data V21.0.csv.,https://bilans-ges.ademe.fr/en/accueil/contenu/index/page/downloaddata/siGras/0
+ff48cb32-52f7-481e-8709-3c4d97d5ea87,Energy,Electricity,electricity-supply_grid-source_nuclear_plant,Electricity supplied from nuclear plant,kWh,0.006,,,,,,,10,3,upstream,ADEME,Base Carbone,2021,2021,,FR,,Climatiq,2021/11/30,Emission intensity for nuclear power plant. The data scope includes upstream emissions. Retrieved from Base Carbone - Data V21.0.csv.,https://bilans-ges.ademe.fr/en/accueil/contenu/index/page/downloaddata/siGras/0
```
Comment (maryarbab) [inline OpenEmissionFactorsDB.csv:154]: source_nuclear would be sufficient.

File: OpenEmissionFactorsDB.csv [hunk 4]
```diff
@@ -151,9 +151,9 @@ c3f5582d-900e-40ca-8bd0-b2bc58964bfb,Energy,Electricity,electricity-supply_grid-
...
-97eb3c1d-98c9-4b25-bf26-ddb74481c913,Energy,Electricity,electricity-energy_source_oil_steam_plant,Electricity supplied from oil steam plant,kWh,0.73,,,,,,,10,2|3,upstream-fuel_combustion,ADEME,Base Carbone,2021,2021,,FR,,Climatiq,2021/08/20,Emission intensity for oil steam plant. The data scope includes upstream and combustion emissions. Retrieved from Base Carbone - Data V21.0.csv.,https://bilans-ges.ademe.fr/en/accueil/contenu/index/page/downloaddata/siGras/0
+97eb3c1d-98c9-4b25-bf26-ddb74481c913,Energy,Electricity,electricity-supply_grid-source_oil_steam_plant,Electricity supplied from oil steam plant,kWh,0.73,,,,,,,10,2|3,upstream-fuel_combustion,ADEME,Base Carbone,2021,2021,,FR,,Climatiq,2021/08/20,Emission intensity for oil steam plant. The data scope includes upstream and combustion emissions. Retrieved from Base Carbone - Data V21.0.csv.,https://bilans-ges.ademe.fr/en/accueil/contenu/index/page/downloaddata/siGras/0
```
Comment (maryarbab) [inline OpenEmissionFactorsDB.csv:156]: source_oil_steam would be sufficient.

File: OpenEmissionFactorsDB.csv [hunk 5]
```diff
@@ -1062,7 +1062,7 @@ ffaacefe-ca92-419c-b301-fc4e8a9c0976,Consumer Goods and Services,Vehicle Mainten
...
+d38cfd0f-ef1d-4054-b343-d4cb20bca45f,Energy,Electricity,electricity-supply_grid-source_transmission_and_distribution,Electricity supplied from grid - transmission and distribution,GBP,not-supplied,1.774,1.536,not-supplied,not-supplied,not-supplied,not-supplied,,3,unknown,BEIS,UK full dataset including conversion factors by SIC code,2019,2019,,GB,,Climatiq,2022/11/23,Emission intensity of supply chain in GBP spend on: electricity/transmission and distribution. Provided by BEIS/DEFRA in the official report on UK and England's carbon footprint to 2019,https://www.gov.uk/government/statistics/uks-carbon-footprint
```
Comment (maryarbab) [inline OpenEmissionFactorsDB.csv:1065]: let's change the activity id to: electricity-supply_grid-source_supplier_mix_losses

File: OpenEmissionFactorsDB.csv [hunk 6]
```diff
@@ -18886,7 +18886,7 @@ fc0ca143-3238-4f68-84c5-819c052110fd,Energy,Electricity,electricity-supply_grid-
...
-11729c8e-2bee-4c53-bd41-289c4987f8ce,Energy,Electricity,electricity-energy_source_imported_coal_fired_plant,Electricity supplied from grid coal-fired power plant (imported coal),kWh,0.885,not-supplied,0.826,0.0016,4e-05,not-supplied,not-supplied,,2|3,upstream-use_phase-transport,GEMIS,GEMIS model and database,2021,2021,2015,DE,,Climatiq,2022/06/20,Emission intensity of large hard-coal-fired power plant for imported coal mix. Free grid feed-in. Total life cycle incl. transport + material input without disposal. Retrieved from GEMIS v5.0.,https://iinas.org/downloads/gemis-downloads/
+11729c8e-2bee-4c53-bd41-289c4987f8ce,Energy,Electricity,electricity-supply_grid-source_imported_coal_fired_plant,Electricity supplied from grid coal-fired power plant (imported coal),kWh,0.885,not-supplied,0.826,0.0016,4e-05,not-supplied,not-supplied,,2|3,upstream-use_phase-transport,GEMIS,GEMIS model and database,2021,2021,2015,DE,,Climatiq,2022/06/20,Emission intensity of large hard-coal-fired power plant for imported coal mix. Free grid feed-in. Total life cycle incl. transport + material input without disposal. Retrieved from GEMIS v5.0.,https://iinas.org/downloads/gemis-downloads/
```
Comment (maryarbab) [inline OpenEmissionFactorsDB.csv:18889]: let's change activity_id to: electricity-supply_grid-source_imported_coal

File: OpenEmissionFactorsDB.csv [hunk 7]
```diff
@@ -18898,20 +18898,20 @@ d0bb725d-d7b3-47ff-8c1e-048a9af2d5fa,Energy,Electricity,electricity-supply_grid-
...
-9d6f97fe-7a61-472e-8d1e-f01b19b3ccf5,Energy,Electricity,electricity-energy_source_natural_gas_ccgt_power_plant,Electricity supplied from natural gas CCGT power plant,kWh,0.402,not-supplied,0.376,0.00072,2e-05,not-supplied,not-supplied,,2|3,upstream-use_phase-transport,GEMIS,GEMIS model and database,2021,2021,2015,DE,,Climatiq,2022/06/20,Emission intensity of large natural gas power plant with gas and steam turbine (combined cycle). Free grid feed-in. Total life cycle incl. transport + material input without disposal. Retrieved from GEMIS v5.0.,https://iinas.org/downloads/gemis-downloads/
+427e718f-3240-48d3-a00f-411df1256419,Energy,Electricity,electricity-supply_grid-source_lignite_fired_power_plant,Electricity supplied from lignite-fired power plant,kWh,1.008,not-supplied,1.0,3e-05,3e-05,not-supplied,not-supplied,,2|3,upstream-use_phase-transport,GEMIS,GEMIS model and database,2021,2021,2015,DE,,Climatiq,2022/06/20,Emission intensity of large lignite-fired power plant (rhenish). Free grid feed-in. Total life cycle incl. transport + material input without disposal. Retrieved from GEMIS v5.0.,https://iinas.org/downloads/gemis-downloads/
```
Comment (maryarbab) [inline OpenEmissionFactorsDB.csv:18903]: let's change activity_id to: electricity-supply_grid-source_lignite

File: OpenEmissionFactorsDB.csv [hunk 8]
```diff
@@ -18898,20 +18898,20 @@ d0bb725d-d7b3-47ff-8c1e-048a9af2d5fa,Energy,Electricity,electricity-supply_grid-
...
+9d6f97fe-7a61-472e-8d1e-f01b19b3ccf5,Energy,Electricity,electricity-supply_grid-source_natural_gas_ccgt_power_plant,Electricity supplied from natural gas CCGT power plant,kWh,0.402,not-supplied,0.376,0.00072,2e-05,not-supplied,not-supplied,,2|3,upstream-use_phase-transport,GEMIS,GEMIS model and database,2021,2021,2015,DE,,Climatiq,2022/06/20,Emission intensity of large natural gas power plant with gas and steam turbine (combined cycle). Free grid feed-in. Total life cycle incl. transport + material input without disposal. Retrieved from GEMIS v5.0.,https://iinas.org/downloads/gemis-downloads/
```
Comment (maryarbab) [inline OpenEmissionFactorsDB.csv:18904]: let's change activity_id to: electricity-supply_grid-source_natural_gas_ccgt

File: OpenEmissionFactorsDB.csv [hunk 9]
```diff
@@ -18898,20 +18898,20 @@ d0bb725d-d7b3-47ff-8c1e-048a9af2d5fa,Energy,Electricity,electricity-supply_grid-
...
+dbbeb059-a0ff-40a8-ae57-6ecfb0b905b4,Energy,Electricity,electricity-supply_grid-source_nuclear_plant,Electricity supplied from nuclear power plant,kWh,0.055,not-supplied,0.052,7e-05,0.0,not-supplied,not-supplied,,2|3,upstream-use_phase-transport,GEMIS,GEMIS model and database,2021,2021,2015,DE,,Climatiq,2022/06/20,Emission intensity of nuclear power plant including intermediate consumption and disposal and uranium for fuel elements from supply mix (with imports). Total life cycle incl. transport + material input without disposal. Retrieved from GEMIS v5.0.,https://iinas.org/downloads/gemis-downloads/
```
Comment (maryarbab) [inline OpenEmissionFactorsDB.csv:18908]: let's change activity_id to: electricity-supply_grid-source_nuclear

File: OpenEmissionFactorsDB.csv [hunk 10]
```diff
@@ -18898,20 +18898,20 @@ d0bb725d-d7b3-47ff-8c1e-048a9af2d5fa,Energy,Electricity,electricity-supply_grid-
...
+619b356f-9ad5-4edc-8303-2fecb5156ec0,Energy,Electricity,electricity-supply_grid-source_power_plant_mix,Electricity supplied from power plant mix,kWh,0.543,not-supplied,0.515,0.00058,4e-05,not-supplied,not-supplied,,2|3,upstream-use_phase-transport,GEMIS,GEMIS model and database,2021,2021,2015,DE,,Climatiq,2022/06/20,Emission intensity of power plant mix. Without power transport/distribution and substation losses. Total life cycle incl. transport + material input without disposal. Retrieved from GEMIS v5.0.,https://iinas.org/downloads/gemis-downloads/
```
Comment (maryarbab) [inline OpenEmissionFactorsDB.csv:18909]: let's change activity_id to: electricity-supply_grid-source_supplier_mix

File: OpenEmissionFactorsDB.csv [hunk 11]
```diff
@@ -18898,20 +18898,20 @@ d0bb725d-d7b3-47ff-8c1e-048a9af2d5fa,Energy,Electricity,electricity-supply_grid-
...
-95fe65d4-e4b9-42fa-bd1e-80873e8a484d,Energy,Electricity,electricity-energy_source_waste_wood_power_plant,Electricity supplied from waste wood power plant,kWh,0.018,not-supplied,0.013,4e-05,1e-05,not-supplied,not-supplied,,2|3,upstream-use_phase-transport,GEMIS,GEMIS model and database,2021,2021,2015,DE,,Climatiq,2022/06/20,Emission intensity of electricity from waste wood power plant (steam turbine) with 20 MW electric without heat utilization. Total life cycle incl. transport + material input without disposal. Retrieved from GEMIS v5.0.,https://iinas.org/downloads/gemis-downloads/
+95fe65d4-e4b9-42fa-bd1e-80873e8a484d,Energy,Electricity,electricity-supply_grid-source_waste_wood_power_plant,Electricity supplied from waste wood power plant,kWh,0.018,not-supplied,0.013,4e-05,1e-05,not-supplied,not-supplied,,2|3,upstream-use_phase-transport,GEMIS,GEMIS model and database,2021,2021,2015,DE,,Climatiq,2022/06/20,Emission intensity of electricity from waste wood power plant (steam turbine) with 20 MW electric without heat utilization. Total life cycle incl. transport + material input without disposal. Retrieved from GEMIS v5.0.,https://iinas.org/downloads/gemis-downloads/
```
Comment (maryarbab) [inline OpenEmissionFactorsDB.csv:18914]: let's change activity_id to: electricity-supply_grid-source_waste_wood

File: OpenEmissionFactorsDB.csv [hunk 12]
```diff
@@ -18898,20 +18898,20 @@ d0bb725d-d7b3-47ff-8c1e-048a9af2d5fa,Energy,Electricity,electricity-supply_grid-
...
+030f9ed0-ecac-420a-984c-e0e545916bd3,Energy,Electricity,electricity-supply_grid-source_natural_gas_combined_cycle_power_plant_100_mw,Electricity supplied from natural gas combined cycle power plant 100 MW,kWh,0.373,not-supplied,0.349,0.00068,2e-05,not-supplied,not-supplied,,2|3,upstream-use_phase-transport,GEMIS,GEMIS model and database,2021,2021,2015,DE,,Climatiq,2022/06/20,Emission intensity of natural gas combined heat and power plant (CHP) 100 MW - energy-related allocated (only effects for CHP electricity). Total life cycle incl. transport + material input without disposal. Retrieved from GEMIS v5.0.,https://iinas.org/downloads/gemis-downloads/
```
Comment (maryarbab) [inline OpenEmissionFactorsDB.csv:18907]: let's change activity_id to: electricity-supply_grid-source_natural_gas_combined_cycle_100_mw

File: OpenEmissionFactorsDB.csv [hunk 13]
```diff
@@ -18898,20 +18898,20 @@ d0bb725d-d7b3-47ff-8c1e-048a9af2d5fa,Energy,Electricity,electricity-supply_grid-
...
-b4a17b92-7ee1-41a6-a277-18e9e074f7e2,Energy,Electricity,electricity-energy_source_hydro_power_plant_10_mw,Electricity supplied from hydro power plant 10 MW,kWh,0.003,not-supplied,0.003,0.0,0.0,not-supplied,not-supplied,,2|3,upstream-use_phase-transport,GEMIS,GEMIS model and database,2021,2021,2015,DE,,Climatiq,2022/06/20,Emission intensity of large run-of-river power plant located in Germany. Total life cycle incl. transport + material input without disposal. Retrieved from GEMIS v5.0.,https://iinas.org/downloads/gemis-downloads/
+b4a17b92-7ee1-41a6-a277-18e9e074f7e2,Energy,Electricity,electricity-supply_grid-source_hydro_power_plant_10_mw,Electricity supplied from hydro power plant 10 MW,kWh,0.003,not-supplied,0.003,0.0,0.0,not-supplied,not-supplied,,2|3,upstream-use_phase-transport,GEMIS,GEMIS model and database,2021,2021,2015,DE,,Climatiq,2022/06/20,Emission intensity of large run-of-river power plant located in Germany. Total life cycle incl. transport + material input without disposal. Retrieved from GEMIS v5.0.,https://iinas.org/downloads/gemis-downloads/
```
Comment (maryarbab) [inline OpenEmissionFactorsDB.csv:18901]: let's change activity_id to: electricity-supply_grid-source_hydro_10_mw

**Review by maryarbab — CHANGES_REQUESTED** [review]

File: OpenEmissionFactorsDB.csv [hunk 14]
```diff
@@ -8592,9 +8592,9 @@ baedd0a5-f273-405b-8c36-c44fd689995b,Energy,Electricity,electricity-supply_grid-
...
+0bd33651-72bd-4b1d-ad84-8cacaf574b5e,Energy,Electricity,electricity-supply_grid-source_operating_margin,Electricity supplied from grid - average operating margin,kWh,not-supplied,0.408,not-supplied,not-supplied,not-supplied,not-supplied,not-supplied,,2,electricity_generation,EMA,Electricity Grid Emissions Factors and Upstream Fugitive Methane Emission Factor,2021,2020,2020,SG,p,https://github.com/digitalronin,2022/04/13,Data published by the Singapore Energy Market Authority (EMA) in 2021. The Grid Emission Factor (GEF) measures the average CO2 emission emitted per unit of electricity. It can be calculated using the Average Operating Margin (OM) or the Build Margin (BM) method. The OM measures the system-wide emissions factor while the BM measures the emissions factor of newer facilities. More details on the GEF could be found in the <Technical Notes> section of the Singapore Energy Statistics (SES) publication.,https://www.ema.gov.sg/statistic.aspx?sta_sid=20140729MPY03nTHx2a1
```
Comment (maryarbab) [inline OpenEmissionFactorsDB.csv:8595]: sorry I missed this and the following one in my previous review: 
electricity-supply_grid-source_supplier_mix_operating_margin
electricity-supply_grid-source_supplier_mix_build_margin

**Review by maryarbab — CHANGES_REQUESTED** [review]

File: OpenEmissionFactorsDB.csv (hunk 14 above)
Comment (gpantelidou) [reply OpenEmissionFactorsDB.csv:8595]: Actually these are the ones I was not sure in my first commit. No problem, let me fix it now so you can review one last time

**Review by maryarbab — APPROVED** [review]

**Pull request merged**